site/docs/foundations/duration.mdx

```diff
@@ -18,67 +20,50 @@ There are four durations available to choose from: instant, perceptible, notable
-
-## Instant (0ms)
+## Instant (0 milliseconds)
 
-An instant duration is an immediate response by the system to a user’s interaction within 0–100ms. The response should take no more than 100ms to complete. Interactions that can trigger an instant response include hovering over a button, clicking a radio button, typing into an editable field, or dragging an object. No feedback is required as the system response itself is the feedback.
+An instant duration is an immediate system response to a user’s interaction within 0–100ms. Users feel they've directly caused the outcome of the interaction. No feedback is required as the system response itself is the feedback. Interactions that can trigger an instant response include clicking a radio button, typing into an editable field, dragging an object, or hovering over a button (pictured below).
 
 <Image
-  src="/img/foundations/duration-instant.svg"
+  src="/img/foundations/duration-instant.png"
   alt="Example of duration instant"
 />
 
-## Perceptible (300ms)
+## Perceptible (300 milliseconds)
 
-A perceptible duration is a short, but detectable delay between a user's action and the system's response.
+A perceptible duration is a short detectable delay between a user's action and the system's response. It’s recommended for revealing secondary guidance or information to the user. For example, tooltips shouldn't appear instantly as the mouse moves over the UI.
 
 <Image
   src="/img/foundations/duration-perceptible.png"
   alt="Example of duration perceptible"
 />
 
-Perceptible duration is recommended for revealing secondary guidance or information to a user.
-
-## Notable (1s)
+## Notable (1 second)
 
-With a notable duration, the user is aware the system is slow to respond. A feeling of loss of control may occur, so it's important the delay is clearly communicated. Notable delays are intentional and may be required to load a data grid or retrieve search results. Should users experience delays longer than 1s they may feel held back by the system.
+Notable delays are intentional and may be required to load or retrieve results. Should users experience delays longer than 1s, they may become frustrated, which can result in them abandoning the action they were trying to take. A 1s delay is usually the maximum limit for experiencing an interaction as one continuous flow. The user understands a ‘notable’ duration as a two-way interaction between them and the system.
 
-A 1s delay is usually the maximum limit for experiencing an interaction as one continuous flow. The user understands a ‘notable’ duration as a two-way interaction between them and the system. By contrast, with an instant (100ms) duration, users feel they've directly caused the outcome of the interaction.
+A progress cursor or [spinner component](go/salt/components/spinner) can be used to prevent users from wondering whether the action they just took didn’t work or is simply taking time to process.
```

Comment:
Link from the same website, doesn't need `go` part, otherwise, public user won't be able to navigate successfully. Also, when go link is actually needed, prefix with `https://`

Reply:
currently a broken link in this form.